### DISCUSSION TITLE
Any way to move compiled .app to /Applications?

### DISCUSSION THREAD

**Discussion by BrushXue** [Homebrew/discussions#2418] [Writing Formulae/Casks]
I want to write a formula to compile `texstudio` for native arm64. Currently there's a [cask](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/texstudio.rb) for x86 only.
Compiling the source code to .app is not a problem. But is there anyway to manage the .app file (installing to `/Applications` folder, and removing?)
Thanks.

**Comment by SMillerDev** [comment]
No, formula can only write to their own install location.
